.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,8 @@
+blank_issues_enabled: true
+contact_links:
+  - name: Rust Programming Language Forum
+    url: https://users.rust-lang.org
+    about: Please ask and answer questions about Rust here.
+  - name: Security Vulnerability
```

Comment:
You don't need to include this link, as GitHub adds it by default (you can see an example in [docs.rs's issue tracker](https://github.com/rust-lang/docs.rs/issues/new/choose)). If you add a `SECURITY.md` in your test repository you should see it there as well.

In our case, the `SECURITY.md` is not on the rustc repo, but [it's shared between all repos in our org](https://github.com/rust-lang/.github).